sections/semantics-embedded-content.include

```diff
@@ -450,7 +450,7 @@ was an English &lt;a href="/wiki/Music_hall"&gt;music hall&lt;/a&gt; singer, ...
     <dd><a>Global aria-* attributes</a></dd>
     <dt><a>DOM interface</a>:</dt>
     <dd>
-      <pre class="idl" data-highlight="webidl" dfn-for="HTMLSourceElement">
+      <pre class="idl" data-highlight="webidl" dfn-for="HTMLSourceElement" id='Picture-HTMLSourceElement'>
```

Comment:
I'll check with Tab on what we can do here. For now just change the `dfn-for=` to `HTMLPictureElement/HTMLSourceElement`